src/app/client/src/app/app.component.ts

```diff
@@ -168,8 +169,9 @@ export class AppComponent implements OnInit, OnDestroy {
     if (this.userService.loggedIn) {
       this.userId = this.userService.userid;
     } else {
-      this.userId = this.userService.anonymousSid;
+      this.userId = this.deviceId;
```

Comment:
Why are we using deviceId as UserId. In this case both did & userID will be same. What is the use? What are we not taking anonymous userId here?